lib/parsers/index.ts

```diff
@@ -339,3 +339,13 @@ export function getTargetFrameworksFromProjectConfig(manifestFile) {
 
   return targetFrameworksResult;
 }
+
+export function getTargetFrameworksFromProjectJson(manifestFile) {
+  const targetFrameworksResult: string[] = [];
+  const frameworks = _.get(manifestFile, 'frameworks', {});
+  for (const framework of Object.keys(frameworks)) {
+    targetFrameworksResult.push(framework);
+  }
```

Comment:
Perhaps just `return Object.keys(frameworks);`?

Reply:
I would even go for `return Object.keys(_.get(manifestFile, 'frameworks', {}));` ;-)